flake.nix

```diff
@@ -116,6 +116,14 @@
             # 0.6.3 in the repo
             refined = hfinal.refined_0_7;
 
+            pretty-simple = hfinal.callHackageDirect
```

Comment:
Woah TIL about `callHackageDirect`, I've been updating `all-cabal-hashes` like a dummy.

Reply:
i can't be bothered to deal with that on slack-web, so I don't. XD

all-cabal-hashes updating is a much better strategy if you have a lot of packages.